### PR TITLE
Signup functionality

### PR DESCRIPTION
**Pull request by anuj-kumary**
- Added signup functionality
- Added logout functionality
Preview :- https://deploy-preview-7--eaglestores-react.netlify.app/

### CODE REVIEW

File: eagle-stores/src/context/auth/auth-context.js [hunk 1]
```diff
@@ -1,39 +1,51 @@
...
         );
-        setUser(resp.data.foundUser);
+        setUser(resp.data.createdUser);
         setToken(resp.data.encodedToken);
-        navigate('/product');
       }
-    } catch (err) {
-      console.log(err);
+    } catch (error) {
+      console.log(error);
```
Comment (varnita619) [inline eagle-stores/src/context/auth/auth-context.js:35]: instead of consoling you can display the error on screen

**Review by varnita619 — APPROVED** [review]
Code is clean and folder structure is good

**Review by hrishi82 — APPROVED** [review]
Things I liked:
1) Really liked the use of &&
2) Modular code

File: eagle-stores/src/context/auth/auth-context.js (hunk 1 above)
Comment (anuj-kumary) [reply eagle-stores/src/context/auth/auth-context.js:35]: Yeah I am going to use Toast for that

**Pull request merged**